olumidae/QuickCredit
# #166103876 Fixes endpoints and add more unit test

**What does this end point do?**
Add fixes to endpoints
Includes login unit tests 

**Description of Task to be completed? **
Fixes the regex pattern from password for sign up endpoint
Includes login unit test and user unit test

**How should this be manually tested?**
After cloning the repo, cd into it and run
Using postman, test the POST /api/v1/auth/signup endpoint
enter password when prompted and ensure password allows string, number and special character

**Any background context you want to provide?**
Not applicable

**What are the relevant pivotal tracker stories?** 
#166103876
